packages/material-ui/src/Select/SelectInput.js

```diff
@@ -105,6 +105,9 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
   };
 
   const handleMouseDown = event => {
+    if (event.button !== 0)
```

Comment:
I'm surprised the CI passes without the bracket. Should we care or ignore it?

Reply:
no lint complain, no problem

Reply:
To expand a bit: The point of a common formatting and linting ruleset is to not care about nitpicks in human code reviews. These discussions always slow down development. They can happen but on a per-ruleset basis not for each and every individual case.

Should we care about single-line brackets: Probably (main argument being diff noise here). But then we should change the rules globally.

Reply:
@eps1lon I couldn't agree more with you

Reply:
I had this rule in mind: https://github.com/airbnb/javascript#blocks--braces. If we agree to enforce them globally (I thought we were), I will add it to a backlog of small changes. It's not really important, I think that it could be handled later, like a Friday evening, to rest.